LoyaltyProgram_Intern/src/main/java/com/pgs/soft/RegisterValidator.java

```diff
@@ -12,12 +12,12 @@
 import com.pgs.soft.service.UserService;
 
 @Component
-public class UserDtoValidator implements Validator {
+public class RegisterValidator implements Validator {
 	
 	private final UserService userService;
 	
 	@Autowired
-	public UserDtoValidator(UserService userService){
+	public RegisterValidator(UserService userService){
```

Comment:
Please stay with convention - if we inject bean by field we should do this everywhere